test/libsolidity/smtCheckerTests/types/array_of_addresses.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.0.0;
+contract C {
+	address[10] a;
+	address payable[10] b;
+
+	function f() public {
+		a = b;
```

Comment:
I can't do that because of:
`TypeError 2271: Built-in binary operator == cannot be applied to types address[10] storage ref and address payable[10] storage ref.`
I did `assert(a[0] == b[0])` isntead.